app/workers/check_organisation_links_worker.rb

```diff
@@ -1,13 +1,14 @@
 # Calls the Link Checker API to verify all links in public editions, either per organisation or not
 class CheckOrganisationLinksWorker
   include Sidekiq::Worker
+  ORGANISATION_EDITION_LIMIT = 25
```

Comment:
I would definitely aim for something more like `1000` here.

Reply:
Sure, was just being conservative - perhaps too much 😄